### PR TITLE
Congyu/restructure

### PR DESCRIPTION
**Pull request by Apollo1840**
@changdiqing  check this out and merge it to the master pls.

### CODE REVIEW

File: backend/studytimeboard/routes.py [hunk 1]
```diff
@@ -246,3 +219,114 @@ def admin_star():
...
+    clean_chart_folder()
+
+
+# depracated
+@app.route('/about')
+def about():
+    return render_template('about.html')
+
+
+# depracated
```
Comment (changdiqing) [inline backend/studytimeboard/routes.py:311]: deprecated

File: backend/studytimeboard/routes.py [hunk 2]
```diff
@@ -246,3 +219,114 @@ def admin_star():
...
+            return {"user": username, "password": password, "is_success": False,
+                    "reason_failure": FlashMessages.REGISTERED_USER}  # todo: change those string to constants
+
+
+@app.route('/api/admin/clean_chart_folder', methods=['GET'])
+def api_admin_clean_chart_folder():
+    clean_chart_folder()
+
+
+# depracated
```
Comment (changdiqing) [inline backend/studytimeboard/routes.py:305]: deprecated

**Review by changdiqing — CHANGES_REQUESTED** [review]
- .pyc files are tmp files and should not be commited
 - stb_app_backend.log: if only serves as logger then is not so important for other developers, you can consider to ignore it as well.

File: backend/studytimeboard/routes.py (hunk 2 above)
Comment (Apollo1840) [reply backend/studytimeboard/routes.py:305]: register 原逻辑暂时保留，以对照其逻辑，对react继续开发。

**Comment by Apollo1840** [comment]
> * .pyc files are tmp files and should not be commited
> * stb_app_backend.log: if only serves as logger then is not so important for other developers, you can consider to ignore it as well.

参照你的上一个commit, 应该是ignore掉了的，没见到相关文件。

**Pull request merged**